administration/src/bp-modules/self-service/CardSelfServiceForm.tsx

```diff
@@ -0,0 +1,133 @@
+import ClearInputButton from '../../cards/extensions/components/ClearInputButton'
+import useWindowDimensions from '../../hooks/useWindowDimensions'
+import BasicDialog from '../../mui-modules/application/BasicDialog'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { ExtensionForm } from '../cards/AddCardForm'
+import { ActionButton } from './components/ActionButton'
+import { IconTextButton } from './components/IconTextButton'
+
+const PrivacyButton = styled('button')`
+  border: none;
+  background-color: transparent;
+  color: blue;
+  text-decoration: underline;
+  padding: 0;
+  cursor: pointer;
+`
+
+const StyledCheckbox = styled(Checkbox)`
+  margin-top: 24px;
+  font-size: 16px;
+  margin-left: 4px;
+`
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: 24px;
+  white-space: pre-line;
+`
+
+const Container = styled('div')`
+  margin-bottom: 24px;
+`
+
+type CardSelfServiceFormProps = {
+  card: CardBlueprint
+  notifyUpdate: () => void
+  dataPrivacyAccepted: boolean
+  setDataPrivacyAccepted: (value: boolean) => void
+  generateCards: () => Promise<void>
+}
+
+const getTooltipMessage = (cardsValid: boolean, dataPrivacyAccepted: boolean): string => {
+  const tooltipMessages: string[] = []
+  if (!cardsValid) {
+    tooltipMessages.push('Mindestens eine Ihrer Angaben ist ungültig.')
+  }
+  if (!dataPrivacyAccepted) {
+    tooltipMessages.push('Bitte akzeptieren Sie die Datenschutzerklärung.')
+  }
+
+  return tooltipMessages.join('\n')
+}
+const CardSelfServiceForm = ({
+  card,
+  notifyUpdate,
+  dataPrivacyAccepted,
+  setDataPrivacyAccepted,
+  generateCards,
+}: CardSelfServiceFormProps): ReactElement => {
+  const { viewportSmall } = useWindowDimensions()
+  const projectConfig = useContext(ProjectConfigContext)
+  const [openDataPrivacy, setOpenDataPrivacy] = useState<boolean>(false)
+  const [openReferenceInformation, setOpenReferenceInformation] = useState<boolean>(false)
+  const cardValid = card.isValid()
+  const cardCreationDisabled = !cardValid || !dataPrivacyAccepted
+  const clearNameInput = () => {
+    card.fullName = ''
+    notifyUpdate()
+  }
+
+  return (
+    <>
+      <Container key={card.id}>
+        <FormGroup label='Name'>
+          <InputGroup
+            large={viewportSmall}
+            placeholder='Erika Mustermann'
+            autoFocus
+            rightElement={
+              <ClearInputButton viewportSmall={viewportSmall} onClick={clearNameInput} input={card.fullName} />
+            }
+            intent={card.isFullNameValid() ? undefined : Intent.DANGER}
+            value={card.fullName}
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              card.fullName = event.target.value
+              notifyUpdate()
+            }}
+          />
+        </FormGroup>
+        {card.extensions.map((ext, i) => (
+          <ExtensionForm key={i} extension={ext} onUpdate={notifyUpdate} viewportSmall={viewportSmall} />
+        ))}
```

Comment:
okay then we should stick to 1980 or what is your suggestion?

Reply:
yes as i already described. this is super difficult to implement at the moment. since we don't have a common mui look and feel for our card creation fields. You have to override super much of the styles. thats why we decided to set a default value as a workaround

Reply:
yes this can be done easily. The issue about that is that then the datepicker starts from today and the user has to scroll probably more than with a default date in 1980.

Reply:
> The issue about that is that then the datepicker starts from today and the user has to scroll

ah I see.
how about using 3 drop downs (day, month, year) instead of the calendar?

Reply:
we can create an improvement task for the birthday. But we have also to ensure that passing date from queryParams works, since this is used by bavaria and nuernberg to create cards. Thats why i wouldn't adjust all this here

Reply:
I think there already is an issue, right?
https://github.com/digitalfabrik/entitlementcard/issues/1672

Reply:
yes and i set a placeholder for now